front/src/modules/pipeline-processes/components/Board.tsx

```diff
@@ -105,6 +109,14 @@ export function Board({
     [board, onUpdate, setBoard],
   );
 
+  const onSelect = (itemKey: string) => {
```

Comment:
We use function if there's no need to have an arrow function like by using a useMemo or a useCallback. 

Also since this is a handle it should begin with handle, handleSelect for example